### PR TITLE
proc: remove dead code

### PR DESCRIPTION
**Pull request by aarzilli**

### CODE REVIEW

File: pkg/proc/variables.go [hunk 1]
```diff
@@ -1063,59 +1060,61 @@ func (v *Variable) structMember(memberName string) (*Variable, error) {
...
-				continue
-			}
-			// Check for embedded field referenced by type name
-			parts := strings.Split(field.Name, ".")
-			if len(parts) > 1 && parts[1] == memberName {
+	queue := []*Variable{v}
+	seen := map[string]struct{}{} // prevent infinite loops
+	first := true
+
+	for len(queue) > 0 {
```
Comment (derekparker) [inline pkg/proc/variables.go:1068]: This changeset seems more like a refactor than dead code elimination. Would you mind sending this patch in separate?

Comment (derekparker) [reply pkg/proc/variables.go:1068]: Ah, it seems you have. I'll take a look at the other PR first then, perhaps we can rebase this after.

Comment (aarzilli) [reply pkg/proc/variables.go]: Yep. I think I messed up a rebase, actually. Fixed it now.

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**